### PR TITLE
python/ansible: skip collection dependency on ansible itself

### PR DESCRIPTION
**Pull request by mtelka**
This will add `community/skydive` and `cyberark/conjur` to the package.

### CODE REVIEW

**Comment by AndWac** [comment]
I have a question regarding system/management/ansible
Both, ansible (ansible-meta.p5m) and ansible-core (ansible.p5m) define this fmri.
Is that intentional? The one from ansible-core is hidden by the one from ansible and during a repo cleanup the one from ansible-core will be deleted.

**Comment by mtelka** [comment]
> I have a question regarding system/management/ansible Both, ansible (ansible-meta.p5m) and ansible-core (ansible.p5m) define this fmri. Is that intentional?

Yes. This was needed for proper (and easy) switch from monolitic ansible package to ansible-core + ansible as we have now.  The  `ansible-core/ansible.p5m` file will not be used again and will be automatically removed by the `python-integrate-project` script once the script is run again.  I was thinking about removing the `ansible-core/ansible.p5m` file in a separate commit, but it is not worth doing (the file is harmless anyway).

> The one from ansible-core is hidden by the one from ansible and during a repo cleanup the one from ansible-core will be deleted.

That's okay.

So in short: do not worry.  The situation will sort out automatically on next ansible-core rebuild/update which should happen shortly (they release new ansible/ansible-core version every month or so).
